### PR TITLE
Disaggregate

### PR DESCRIPTION
**Pull request by michaeldorman**
disaggregate

### CODE REVIEW

File: 05-geometry-operations.qmd [hunk 1]
```diff
@@ -502,7 +505,155 @@ shapely.geometry.MultiPoint(polygon.exterior.coords)
 
 ### Aggregation and disaggregation
 
-...
+Raster datasets can also differ with regard to their resolution.
```
Comment (Robinlovelace) [inline 05-geometry-operations.qmd:508]: Not only can they, they do. How about something like

> Raster datasets vary based on their resolution, from high resolution datasets that enable individual trees to be seen, to low resolution datasets covering the whole Earth.

File: 05-geometry-operations.qmd [hunk 2]
```diff
@@ -502,7 +505,155 @@ shapely.geometry.MultiPoint(polygon.exterior.coords)
 
 ### Aggregation and disaggregation
 
-...
+Raster datasets can also differ with regard to their resolution. 
+To match resolutions, or to modify the resolution for other reasons (such as conserving storage space), one can either decrease (aggregate) or increase (disaggregate) the resolution of one raster.
```
Comment (Robinlovelace) [inline 05-geometry-operations.qmd:509]: Minor suggested tweak:

```
Raster datasets can be transformed to either decrease (aggregate) or increase (disaggregate) their resolution for a number of reasons.
Aggregation can reduce computational resource requirements of raster storage and subsequent steps, disaggregation can be used to match other datasets or to add detail.
```

File: 05-geometry-operations.qmd [hunk 3]
```diff
@@ -502,7 +505,155 @@ shapely.geometry.MultiPoint(polygon.exterior.coords)
 
 ### Aggregation and disaggregation
 
-...
+Raster datasets can also differ with regard to their resolution. 
+To match resolutions, or to modify the resolution for other reasons (such as conserving storage space), one can either decrease (aggregate) or increase (disaggregate) the resolution of one raster. 
+As an example, we here change the spatial resolution of `dem.tif` by a factor of 5 (Figure @fig-raster-aggregate). 
+
+Raster aggregation is, in fact, a special case of raster resampling (see @sec-raster-resampling), where the target raster grid is aligned with the original raster, only with coarser pixels. Raster resampling, is the general case where the new grid is not necessarily an aggregation of the original one, but any other case as well (such as a rotated and/or shifted one, etc.).
```
Comment (Robinlovelace) [inline 05-geometry-operations.qmd:512]: :+1:

**Review by Robinlovelace — APPROVED** [review]
This is great content, enjoying the read! Minor suggested edits, feel free to accept/reject/tweak as appropriate.

**Comment by michaeldorman** [comment]
> This is great content, enjoying the read! Minor suggested edits, feel free to accept/reject/tweak as appropriate.

Thanks @Robinlovelace !

File: 05-geometry-operations.qmd (hunk 1 above)
Comment (michaeldorman) [reply 05-geometry-operations.qmd:508]: Great! Added it

File: 05-geometry-operations.qmd (hunk 2 above)
Comment (michaeldorman) [reply 05-geometry-operations.qmd:509]: Great!

**Pull request merged**